src/readii/feature_extraction.py

```
@@ -1,392 +1,413 @@
+	negativeControl : str
+		Name of negative control to generate from the CT to perform feature extraction on. If set to None, will extract features from original CT image.
+	randomSeed : int
+		Value to set random seed with for negative control creation to be reproducible.
+
+	Returns
+	-------
+	OrderedDict[Any, Any]
+		Dictionary containing image metadata, versions for key packages used for extraction, and radiomic features
+	"""
+	# If no pyradiomics paramater file passed, use default
+	if pyradiomicsParamFilePath == None:
```

Ruff (0.8.0)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)